plugins/woocommerce/woocommerce.php

```diff
@@ -54,7 +54,7 @@ function WC() { // phpcs:ignore WordPress.NamingConventions.ValidFunctionName.Fu
  * @since  4.4.0
  * @return \Psr\Container\ContainerInterface The WooCommerce PSR11 container.
  */
-function wc_get_container() : \Psr\Container\ContainerInterface {
+function wc_get_container() {
```

Comment:
Could a return type (`Automattic\WooCommerce\Container`) still be useful or, if there is a reason not to, should we update the stated type in the `@return` tag (I don't think it's strictly accurate, if I'm reading correctly)?

Reply:
I don't think we really need a return type here (most legacy WooCommerce functions don't have anyway), but indeed, the docblock needs fixing, thanks.